docs/source/markdown/podman-pod-create.1.md

```diff
@@ -152,6 +152,19 @@ podman generates a UUID for each pod, and if a name is not assigned
 to the container with **--name** then a random string name will be generated
 for it. The name is useful any place you need to identify a pod.
 
+#### **--userns**=*mode*
+
+Set the user namespace mode for all the containers in a pod. It defaults to the **PODMAN_USERNS** environment variable. An empty value ("") means user namespaces are disabled.
+
+Valid _mode_ values are:
+
+- **auto[:**_OPTIONS,..._**]**: automatically create a namespace. It is possible to specify these options to `auto`:
```

Comment:
Unless I'm misinterpreting https://raw.githubusercontent.com/containers/podman/main/docs/MANPAGE_SYNTAX.md, I think these and the ones following should have a single and not a double asterisk

```suggestion
- *auto[:*_OPTIONS,..._*]*: automatically create a namespace. It is possible to specify these options to `auto`:
```

Reply:
I am not sure, but all over the doc options like these are in bold vs italics (which is what single asterisk does). I do think the bold is easier to read than the italics.

Reply:
Fixed